src/backend/renderer/gles2/mod.rs

```diff
@@ -454,20 +452,18 @@ extern "system" fn gl_debug_log(
     _severity: ffi::types::GLenum,
     _length: ffi::types::GLsizei,
     message: *const ffi::types::GLchar,
-    user_param: *mut nix::libc::c_void,
+    _: *mut nix::libc::c_void,
```

Comment:
This could use a span for the gles renderer?

Reply:
This is a non-issue, as the `Gles2Renderer` is already setting up the span, thus every GL-debug-callback will still have the span active. (I will check, that `with_context` does do that, to make sure even Raw GL commands will have it).